hashicorp/boundary-ui
# ICU-13883 add remove org modal

✅ Closes: https://hashicorp.atlassian.net/browse/ICU-13883

# Description

Add a modal for when a user deselects a singular org with projects. 
Add a separate modal for when a user deselects all and orgs have projects attached to them.




## Screenshots (if appropriate)


https://github.com/user-attachments/assets/8ca51091-58c6-4320-af79-fc88239622d6



## How to Test

1. Make sure you have an org created that has various projects
2. Go to roles page -> click on a role
3. Click "manage" dropdown -> "manage scopes" -> "manage custom scopes"
4. For the Org with many projects click the link under "Projects selected"
5. Select and save a couple projects.
6. After saving, you should now be on the "manage custom scopes" page again.
7. If you deselect the org with projects then the modal should appear.
8. click "Remove org and projects" and now project count should be zero
9. Click "save" and scopes tab table should not have those projects

## Checklist
